packages/specification/prescriptions-api.yaml

```diff
@@ -0,0 +1,269 @@
+    | 401           | ACCESS_DENIED         | User does not have permission for a particular request.                                                                               |
+    | 403           | FORBIDDEN             | Insufficient authorization.                                                                                                           |
+    | 404           | NOT_FOUND             | Route not found.                                                                                                                      |
+    | 500           | INTERNAL SERVER ERROR | The server has encountered a situation it does not know how to handle.                                                                |
+    | 504           | GATEWAY_TIMEOUT       | If your request has not been fully processed within 59 seconds, you will get this response. Retry the request with identical payload. |
+
+    For further details on common error codes, see [HTTP status codes](https://digital.nhs.uk/developer/guides-and-documentation/reference-guide#http-status-codes)
+
+x-spec-publication:
+  operation-order:
+    - operations:
+        - method: POST
+          path: /GetMyPrescriptions/Bundle
+
+paths:
+  /GetMyPrescriptions/Bundle:
+    get:
+      operationId: get-my-prescriptions-bundle
+      summary: Get prescriptions for the given user
+      description: |
+        ## Overview
+        Use this endpoint to get a Bundle containing a list of the user's prescriptions.
+      parameters:
+        - $ref: "#/components/parameters/BearerAuthorisation"
+        - $ref: "#/components/parameters/NhsNumber"
+        - $ref: "#/components/parameters/RequestID"
+      responses:
+        "200":
+          description: Successful retrieval.
+          content:
+            application/fhir+json:
+              schema:
+                $ref: "#/components/schemas/bundle-response"
+              examples:
+                prescriptions-present:
+                  description: A successful response to a GetMyPrescriptions/Bundle request, containing one or more prescriptions.
+                  value:
+                    $ref: examples/GetMyPrescriptions/Bundle/success.json
+                prescriptions-not-present:
+                  description: A successful response to a GetMyPrescriptions/Bundle request, where no prescriptions were found.
+                  value:
+                    $ref: examples/GetMyPrescriptions/Bundle/success-empty.json
+        "4XX":
+          description: Invalid request.
+          content:
+            application/fhir+json:
+              schema:
+                $ref: "#/components/schemas/operation-outcome"
+              examples:
+                example:
+                  description: |
+                    An error response to a GetMyPrescriptions/Bundle request.
+                    Resource not found for the given NHS number.
```

Comment:
This is the same as the success with zero Prescriptions above?

Reply:
This one is an OperationOutcome, whereas the empty one is still the Bundle, is that not what we're after?

Reply:
Yes, different outputs, but what I meant was I _think_ the scenarios described are the same? Definitely not a biggy!

Reply:
Leaving this as-is.